.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+repos:
+  - repo: local
+    # FIXME: use anchors
+    hooks:
+      - id: coding-style-js
+        name: Coding style js
+        description: Check if code respects the javascript coding-style
+        language: system
+        entry: ./scripts/eslint.sh
+        stages: ["commit"]
```

Comment:
why not just using https://github.com/pre-commit/mirrors-eslint ?

Reply:
Having a script for each lint is useful to share them between the CI and the pre-commit.

We could, as you said, use this link and instead of doing 4 Dockerfiles, just one that runs the pre-commit, but again, I think it adds complexity in the architecture and is even maybe bad practise.